CoreNotesView/src/au/gov/asd/tac/constellation/views/notes/NotesViewPane.java

```diff
@@ -160,6 +168,40 @@ public void onChanged(final ListChangeListener.Change event) {
             }
         });
 
+        notesViewEntries.forEach(entry -> {
+            if (!entry.isUserCreated()) {
+
+                final List<String> tags = entry.getTags();
+                for (final String tag : tags) {
+                    if (!tagsUpdater.contains(tag)) {
+                        tagsUpdater.add(tag);
+                    }
+                }
+            }
+        });
+
+        tagsFiltersList = FXCollections.observableArrayList(tagsUpdater);
+
+        // CheckComboBox for the Auto Note filters
+        autoFilterCheckComboBox = new CheckComboBox(tagsFiltersList);
+        autoFilterCheckComboBox.setStyle(String.format("-fx-font-size:%d;", FontUtilities.getApplicationFontSize()));
+        autoFilterCheckComboBox.getCheckModel().getCheckedItems().addListener(new ListChangeListener() {
```

Comment:
convert to a lambda (looks like you'll need to specify the type of the parameter though due to ambiguity)